src/App.vue

```diff
@@ -1,36 +1,82 @@
 <template>
   <div id="app">
-    4.3
+    <SearchPage
+      v-if="selectedMovie === null"
```

Comment:
I'd make computed property `isAnyMovieSelected` and probably revert logic
```
<MoviePage v-if="isAnyMovieSelected" />
<SearchPage v-else/>
```
would be easier to read, don't like logic in the templates